tests/integration/GKE.md

```diff
@@ -77,7 +78,7 @@ gcloud container clusters \
 
 ```bash
 gcloud container clusters get-credentials ${CLUSTER_NAME} \
-   --zone ${ZONE} --project ${PROJECT_NAME}
+   --zone ${ZONE} --project ${PROJECT_ID}
```

Comment:
The documentation in gcloud says that it's the project name, not the ID.

Reply:
It doesn't work with project name, it only works with project_ID

Reply:
Interesting ... I was sure I used name in the past.  Whatever ... just delete the name and update example above.